saas/fsm/unsanctioned_apps_with_contract/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## v2.0
+
+- Updated Credential service and Endpoint
+
+## v1.1
+
+- Updated the metadata
+
+## v1.0
+
+- initial release
```

Comment:
Isn't this the initial release

Reply:
it is.  i just copied the changelog and forgot to update :)